src/main/scala/me/rexim/morganey/ReplAutocompletion.scala

```diff
@@ -72,13 +73,18 @@ class ReplAutocompletion(globalContext: () => InterpreterContext) extends Comple
         .map { case (root, f) => (root, stripExtensionIfModuleFile(f)) }
         .filter { case (root, f) => fileNameFilter(f.getName) }
         .map(relativize)
+        .map(fixRelativePathForWindows)
         .map(relativeFileToLoadPath)
 
     def relativize(baseAndFile: (File, File)): String = {
       val (base, file) = baseAndFile
       base.toURI().relativize(file.toURI()).getPath()
     }
 
+    def fixRelativePathForWindows(path: String): String =
+      if (runningOnWindows) path.replace('/', File.separatorChar)
```

Comment:
Nah, after thorough research I've decided that the current implementation is okay and it's generally not a good idea to use forward slashes (aka `/`) on Windows.

In most of the cases it will work, but they say that forward slashes won't work with UNC paths. And that's not a very nice if we couldn't load Morganey modules from UNC path, because people are using it actually (for example, UNC paths could be used as a workaround for known long path problem on Windows).

So, we're keeping the current implementation.